sample/sample-with-di/app/src/main/java/com/dropbox/kaiken/sample/advancedsample/app/SplashActivity.kt

```diff
@@ -29,7 +29,7 @@ class SplashActivity : AppCompatActivity(), AuthOptionalActivity {
         val navController = navHostFragment.navController
 
         // Setup bottom navigation with navigation controller
-        bottomNavigationView.setupWithNavController(navController)
+        findViewById<BottomNavigationView>(R.id.bottomNavigationView).setupWithNavController(navController)
```

Comment:
I didn't implement viewbinding since it is just a sample app.  Open to doing it if someone really cares :)